terraform/file_scanning.tf

```diff
@@ -1,16 +1,14 @@
 module "s3_scan_objects" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
+  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.1"
 
-  product_name          = "share-files-securely"
   s3_upload_bucket_name = module.share_files_securely_bucket.s3_bucket_id
 
   billing_tag_value = var.billing_code
 }
 
 module "s3_scan_objects_dev" {
-  source = "github.com/cds-snc/terraform-modules?ref=v3.0.9//S3_scan_object"
+  source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.1"
 
-  product_name                  = "share-files-securely-dev"
```

Comment:
You can use the new suffix here so things install properly:
```hcl
scan_queue_suffix = "DEV"
```

Reply:
Done! Thank you for making the change to the base module and for noticing this.